### DISCUSSION TITLE
ClickOnce - Could not load file or assembly CefSharp.Core.Runtime

### DISCUSSION THREAD

**Discussion by A6932401** [cefsharp/CefSharp#3771] [Show and tell]
Dear Team,

Could not load file or assembly 'CefSharp.Core.Runtime, Version=92.0.260.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The system cannot find the file specified.

My Code:
 Cef.EnableHighDPISupport();
                browser = new ChromiumWebBrowser("https://xxx.com/");
                pictureBox1.Controls.Add(browser);

I  Develop  in VS 2015.

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp.MinimalExample for a working example. 

> I Develop in VS 2015.

VS2017 or VS2019 are recommend, no testing is done with VS2015

**Reply by A6932401** [reply]
Error Came from Existing windows Clickonce Application.

If, I Create New project and add ChromiumWebBrowser its working Fine.

We need any Changes For Existing Clickonce application?

**Reply by amaitland** [reply]
See https://github.com/cefsharp/CefSharp/discussions/3613#discussioncomment-842336

No testing has been done with VS2015.

**Reply by A6932401** [reply]
Same Error came in VS 2017 Also.

![Screenshot 2021-08-25 100110](https://user-images.githubusercontent.com/73706751/130726834-36c4348c-ec9c-4e07-a7b0-a6954c3b1987.png)

**Reply by amaitland** [reply]
Only targeting `x64` or `x86` are currently supported, currently `AnyCPU` doesn't work.

If you are still having errors then you'll need to provide more detail, I don't know what you did exactly.

**Reply by amaitland** [reply]
Check your `ClickOnce` installation folder and confirm the files listed in https://github.com/cefsharp/CefSharp/wiki/Output-files-description-table-%28Redistribution%29 were installed.

**Reply by A6932401** [reply]
Any Option Available  for Target ANYCPU?

Working Fine while use X86 Target.

**Reply by amaitland** [reply]
> Any Option Available for Target ANYCPU?

This is a known issue as per https://github.com/cefsharp/CefSharp/issues/3319#issuecomment-791783346

You can try self host the browser sub process. https://github.com/cefsharp/CefSharp/blob/cefsharp/93/CefSharp.Core/BrowserSubprocess/SelfHost.cs

**Comment by amaitland** [comment]
For anyone that ends up here I've created a more detailed guide for publishing using `ClickOnce` see https://github.com/cefsharp/CefSharp/discussions/4056
